VocaDbWeb/Scripts/ViewModels/User/FollowedArtistsViewModel.ts

```diff
@@ -35,7 +39,8 @@ module vdb.viewModels.user {
 		public page = ko.observableArray<dc.RatedSongForUserForApiContract>([]); // Current page of items
 		public paging = new ServerSidePagingViewModel(20); // Paging view model
 		public pauseNotifications = false;
-		public resources = ko.observable<any>();
+        public resources = ko.observable<any>();
```

Comment:
Could you correct all these indentation changes? Thanks.

Reply:
fixed